docs/UserGuide.md

```diff
@@ -394,12 +394,16 @@ Parameters:
 in the filtered contact list must have part of the `[term]` in the `[field]`
 specified above.
 * `[taken]` (optional). Only for filtering by `course`, this parameter additionally filters
-  out contacts who have finished the course.
+  out contacts who have finished/haven't started the course as of the current date of your system.
 
     List of options:
 
-    * `true` - Remove contacts that have finished taking the course.
-    * `false` - (default). Include contacts who have finished taking the course.
+    * `true` - Remove contacts whose courses' start and end date range are outside 
+    your system's current date.
```

Comment:
Shouldn't the phrasing for this part be the same as above (line 397), and also this phrasing may sound a bit technical.